src/core/game.vala

```diff
@@ -2,6 +2,8 @@
 
 public interface Games.Game : Object {
 	public abstract string name { get; }
+	public abstract string name_complete { get; }
+	public abstract int disc_number { get; set; }
```

Comment:
Actually this doesn't look like something we want, _and_its not used, two good reasons not to have it. :p